dbflute-engine/src/main/java/org/dbflute/DfBuildProperties.java

```diff
@@ -333,4 +341,14 @@ public Integer getVersion() { // means DBFlute's version
     public void setVersion(Integer version) {
         this._version = version;
     }
-}
\ No newline at end of file
+
+    public String getWorkDir() {
+        // default value is current directory of Java execution,
+        // and it is basically DBFlute client directory (in official DBFlute process)
+        return _workDir == null ? "./" : _workDir;
```

Comment:
Is the default value "./" already unneeded for new specification?
I think this logic should be simple.
```
return _workDir; // null allowed
```